src/docs/get-started/install/_get-sdk-win.md

```diff
@@ -20,32 +20,33 @@ To update an existing version of Flutter, see [Upgrading Flutter](/docs/developm
 
 ### Update your path
 
-If you wish to run Flutter commands in the regular Windows command prompt, take
+If you wish to run Flutter commands in the regular Windows console, take
```

Comment:
I don't have a Windows 10 box handy.  Did they rename "Command Prompt" to "Console" in the Start Menu?  If so, ok.  If not, then I think you should leave this alone.

Reply:
See comments below. In this case, `console` is more accurate because there are many Start menu icons that could lead to the Windows console. And more obvious since the alternative is the `Flutter Console`.